test/_init_tests.js

```diff
@@ -160,6 +166,10 @@ class TestConfig {
     return this.if(this.platform.isIos.bind(this.platform));
   }
 
+  ifIe() {
+    return this.if(this.platform.isIe.bind(this.platform));
```

Comment:
Aha, the lists are independent, so we need to remove it from `skipMatchers` before `ifIe` kicks in. Done.